config/webpack.config.js

```diff
@@ -44,6 +44,7 @@ const commonConfig = ({ dev }) => {
       publicPath,
       chunkFilename: dev ? '[name].js' : '[name].[contenthash].js',
     },
+    devtool: process.env.SENTRY_AUTH_TOKEN ? 'source-map' : false,
```

Comment:
This needs to be `hidden-source-map` so there is no reference in the source code and we do not load sour maps via Akamai.

Reply:
I dont have perms yet but the goal would be to build the source maps in the CI/CD set up ,send to sentry every time theres a merge, and then delete it, so the build never makes it to akamai/served to customers.
Similar to this: https://github.com/RedHatInsights/insights-advisor-frontend/blob/cdd23204e7285353094ef1333c12efc13f8983b3/.github/workflows/webpack.yml

Reply:
> and then delete it,

The problem is, that the source map is still referenced in the source file, and the browser will fire a request. The fact the response will be a 404 is even worse than if it was a 200. It will always skip the cache nodes (because the asset does not exist there) and will go directly to our frontend containers.

We don't want Chrome to reference the source maps at all.

Reply:
I see. Making adjustments

Reply:
It can't be in the build step which is publishing the container. That is the hard limit.

Reply:
This has been updated to use hidden-source-maps instead